drivers/seu/Kconfig

```
@@ -0,0 +1,25 @@
+# Copyright (c) 2023 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+# Single Event Upset(SEU) configuration options
+
+menuconfig SEU
+	bool "Single Event Upset"
+	depends on ARM_SIP_SVC_SUBSYS
+	help
+	  SEU is the change in state of a storage element inside a device or system.
```

this probably wants to be marked as experimental

```suggestion
menuconfig SEU
	bool "Single Event Upset [EXPERIMENTAL]"
	depends on ARM_SIP_SVC_SUBSYS
	select EXPERIMENTAL
	help
	  SEU is the change in state of a storage element inside a device or system.
```